services/QuillLMS/app/views/application/_header.html.erb

```diff
@@ -1,16 +1,50 @@
 <% current_path = request.env['PATH_INFO'] %>
 <% on_sign_up_or_log_in = ['account/new', '/session', 'password_reset', 'sign-up', 'finish_set_up'].any? { |str| current_path.include?(str)} %>
+<% desktop_navbar_class_name = on_sign_up_or_log_in ? "non-standard-navbar navbar desktop-navbar" : "navbar desktop-navbar" %>
+<% mobile_navbar_class_name = on_sign_up_or_log_in ? "non-standard-navbar navbar mobile-navbar" : "navbar mobile-navbar" %>
 
-<% navbar_class_name = on_sign_up_or_log_in ? "non-standard-navbar navbar" : "navbar" %>
 
 <% if show_site_navigation? %>
-  <nav class="<%=navbar_class_name%>" role="navigation">
-  	<div class="container">
-  		<%= render partial: 'sub_header' %>
-  	</div>
-  </nav>
+	<nav class="<%=desktop_navbar_class_name%>" role="navigation">
+		<%= render partial: 'sub_header' %>
+	</nav>
+	<nav class="<%=mobile_navbar_class_name%>" role="navigation">
+		<%= render partial: 'mobile_sub_header' %>
+	</nav>
 <% end %>
 
 <% if show_site_navigation? && current_path.include?('cms') %>
-	<div class="navbar-divider-bar green"></div>'
+	<div class="navbar-divider-bar green"></div>
 <% end %>
+
+<script>
+	function toggleMobileSecondaryDropdown(primaryDropdownOpen) {
+    const dropdown = document.querySelector('#mobile-subnav-dropdown');
+
+		if(!dropdown) { return }
+
+    const parent = dropdown.parentElement
+
+    if(primaryDropdownOpen) {
+      parent.classList.remove('open');
+      return;
```

Comment:
I think it should be safe to make this one line and just call `return parent.classList.remove('open');` since it doesn't look like it matters what value comes back from this function.  I could be wrong, though.